fdbrpc/ReplicationUtils.cpp

```diff
@@ -26,6 +26,12 @@
 #include "fdbrpc/Replication.h"
 
 
```

Comment:
I would suggest referencing [ILogSystem's comments](https://github.com/apple/foundationdb/blob/d41852abbc3747646d6bf7124c9bea828badcb8b/fdbserver/LogSystem.h#L640) as the shining star example of API comments that have actually been remarkably useful to reference, because they
(1) explain in context what each of the arguments represent
(2) succinctly detail the contracts around when one may or should call the function
(3) explain the meaning of the return value, any postconditions, and what failures may happen instead

The rest of my comments left below this line are basically questions that relate to one of these three things.